x-pack/test/functional/apps/lens/group2/table.ts

```diff
@@ -36,7 +36,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect(await lens.getDatatableCellText(0, 0)).to.eql('169.228.188.120');
       // Remove the sorting
       await retry.try(async () => {
-        await lens.changeTableSortingBy(0, 'none');
+        await lens.changeTableSortingBy(0, 'descending');
```

Comment:
Why this change?

Reply:
Had a look at the code underneath this change: apparently the new datagrid does mark any more the current sorting button as selected on the column actions dropdown.
So right now there's no way to get which "sorting" mode is currently "active" other than looking at the presence of the icon on the column.
Before a `--selected` class was added to the item.

Is this intentional?

Reply:
Yep, I removed `.euiDataGridHeader__action--selected` from our classNames/CSS as it wasn't actually applying any useful styles anymore, and I hadn't realized this FTR test was hooking into it 😬 I replaced it instead with logic that adds "Sort/Unsort" text to represent currently sorted state:

<img width="155" alt="" src="https://github.com/user-attachments/assets/b613c51d-4fe5-4abd-97ba-d5e4fcb8ba50">

But it looks like the Lens table completely overrides the sort label with your own `Sort ascending/descending` node, which means that the default "Unsort" text can't be hooked into.

Options I can look into:

- Remove your `label` override with a `schemaLabel` instead, which means that you get EuiDataGrid's "Unsort" text (the `schemaLabel` replaces the `A-Z/Z-A` text only)
- Re-add a selected state as a `data-` attribute of some kind that you can hook into (would likely be the next release, and I can revert the Lens changes at that point)